### ISSUE TITLE
invalid-first-character-of-tag-name error in {{ }}

### ISSUE THREAD

**Issue by armano2** [mysticatea/vue-eslint-parser#32]
copy of vuejs/eslint-plugin-vue#590

**What did you do? Please include the actual source code causing the issue.**

```html
<span>{{ content.replace(/<[^>]+>/g,"") }}</span>
```

**What did you expect to happen?**

There should not be an error for the RegExp code in {{ }}

**What actually happened? Please include the actual, raw output from ESLint.**

```
invalid-first-character-of-tag-name error in {{ }}
```

**Comment by mysticatea** [comment]
It's the correct result. See https://html.spec.whatwg.org/multipage/parsing.html#parse-error-invalid-first-character-of-tag-name.
